content/en/docs/reference/using-api/_index.md

```diff
@@ -50,27 +50,29 @@ Here's a summary of each level:
 
 - Alpha:
   - The version names contain `alpha` (for example, `v1alpha1`).
-  - The software may contain bugs. Enabling a feature may expose bugs. A
-    feature may be disabled by default.
-  - The support for a feature may be dropped at any time without notice.
+  - Alpha API versions are disabled by default.
+  - The software may contain bugs. Enabling a feature may expose bugs.
+  - Support for an alpha API may be dropped at any time without notice.
   - The API may change in incompatible ways in a later software release without notice.
   - The software is recommended for use only in short-lived testing clusters,
     due to increased risk of bugs and lack of long-term support.
 
 - Beta:
   - The version names contain `beta` (for example, `v2beta3`).
+  - Beta API versions are typically disabled by default.
+  - Beta API versions have a maximum lifetime of 3 minor releases from introduction
+    to deprecation, and 3 minor releases from deprecation to removal.
   - The software is well tested. Enabling a feature is considered safe.
-    Features are enabled by default.
   - The support for a feature will not be dropped, though the details may change.
 
   - The schema and/or semantics of objects may change in incompatible ways in
-    a subsequent beta or stable release. When this happens, migration
-    instructions are provided. Schema changes may require deleting, editing, and
-    re-creating API objects. The editing process may not be straightforward.
+    a subsequent beta or stable API version. When this happens, migration
+    instructions are provided. Adapting to a subsequent beta or stable API version
+    may require editing or re-creating API objects, and may not be straightforward.
     The migration may require downtime for applications that rely on the feature.
   - The software is not recommended for production uses. Subsequent releases
-    may introduce incompatible changes. If you have multiple clusters which
-    can be upgraded independently, you may be able to relax this restriction.
+    may introduce incompatible changes. Use of beta API versions will eventually
```

Comment:
nit: Could you be more specific instead of suggesting future consequences,
`will eventually be required`

Reply:
I think it's pretty clear that this follows from the max lifetime mentioned a few lines above.

Reply:
tried to be more concrete here